%YAML:1.0
---
CameraExtrinsicMat: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [ -2.7154649366318617e-02, 6.9791520467871682e-02,
       9.9719194174871628e-01, 1.7766969639134028e-01,
       -9.9960204787364826e-01, 5.7284455043832283e-03,
       -2.7621201981666177e-02, 4.4606650071559874e-02,
       -7.6400853791683265e-03, -9.9754515115000486e-01,
       6.9608192854814566e-02, -2.9598809357287270e-01, 0., 0., 0., 1. ]
CameraMat: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 6.0890519134323574e+02, 0., 3.3768785597691641e+02, 0.,
       6.0789426622496728e+02, 2.5268039024691987e+02, 0., 0., 1. ]
DistCoeff: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 6.4586182712311258e-02, 4.4142000068489917e-01,
       -2.6232608244149175e-03, 3.4697168137134094e-03,
       -2.1304339392265965e+00 ]
ImageSize: [ 640, 480 ]
ReprojectionError: 3.0574737521091438e-01
ChessboardViewNUM: 25
